database/data_gen/dataGeneration.js

```diff
@@ -12,6 +12,28 @@ const sidebarPath = path.join(`${__dirname}/sidebarData.csv`);
 
 const weightedTrueGenerator = (percentageChance) => Math.random() * 100 < percentageChance;
 
+const course1 = {
```

Comment:
Was this created for test purposes (like insert one object first)? If you can seed from course 1, I think you can have it removed and refactored the data gen function.